cargo-insta/src/cli.rs

```diff
@@ -265,11 +265,21 @@ fn query_snapshot(
             style("a").green().bold(),
             style("keep the new snapshot").dim()
         );
-        println!(
-            "  {} reject     {}",
-            style("r").red().bold(),
-            style("keep the old snapshot").dim()
-        );
+
+        if old.is_some() {
+            println!(
+                "  {} reject     {}",
+                style("r").red().bold(),
+                style("keep the old snapshot").dim()
```

Comment:
feel free to ignore, but this would maintain the alliteration of `r`!

```suggestion
                style("retain the old snapshot").dim()
```